homework/CSharpOOP/Program.cs

```diff
@@ -0,0 +1,133 @@
+//var totalsecond = secondNumber * secondPrice - ((secondNumber * secondPrice) * discount / 100);
+//var totalthird = thirdNumber * thirdPrice - ((thirdNumber * thirdPrice) * discount / 100);
+//var firstMetod = new Homework_1().SeveralItems(firstItem, firstPrice, discount, 5);
+//var secondMetod = new Homework_1().SeveralItems(secondItem, secondPrice, discount, 5);
+//var thirdMetod = new Homework_1().SeveralItems(thirdItem, thirdPrice, discount, 5);
+//var totalPrice = totalfirst + totalsecond + totalthird;
+//Console.WriteLine(firstMetod);
+//Console.WriteLine(secondMetod);
+//Console.WriteLine(thirdMetod);
+//Console.WriteLine($"Итого к оплате: {totalPrice}");
+#endregion
+#region Task_3
+
+//var optionalFirst = new Homework_1().SeveralItems(firstItem, firstPrice, discount, 5);
+//var optionalSecond = new Homework_1().SeveralItems(secondItem, secondPrice, discount, 5);
+//var optionalThird = new Homework_1().SeveralItems(thirdItem, thirdPrice, discount, 5);
+//var requiredFirst = new Homework_1().SeveralItems(firstItem, firstNumber, firstPrice, discount);
+//var requiredSecond = new Homework_1().SeveralItems(secondItem, secondNumber, secondPrice, discount);
+//var requiredThird = new Homework_1().SeveralItems(thirdItem, thirdNumber, thirdPrice, discount);
+#endregion
+#region Task_4
+//Console.WriteLine("Введите длину первого катета");
+//var firstLeg = double.Parse(Console.ReadLine());
+//Console.WriteLine("Введите длинну второго катета");
+//var secondLeg = double.Parse(Console.ReadLine());
+//Console.WriteLine("Введите высоту");
+//var height = double.Parse(Console.ReadLine());
+//var Area = new Homework_1().Area(firstLeg, secondLeg, height, out double triangleArea, out double quadrilateralArea, out double pentagonArea);
+//Console.WriteLine($"Площадь треугольника: {triangleArea}");
+//Console.WriteLine($"Площадь четырехугольника: {quadrilateralArea}");
+//Console.WriteLine($"Площадь пятиугольника: {pentagonArea}");
+
+#endregion
+#region Task_5
+//Console.WriteLine("Введите длину первого катета");
+//var firstLeg = int.Parse(Console.ReadLine());
+//Console.WriteLine("Введите длинну второго катета");
+//var secondLeg = int.Parse(Console.ReadLine());
+//Console.WriteLine("Введите радиус");
+//var radius = double.Parse(Console.ReadLine());
+//double pi = Math.PI;
+//var areaTriangl = new Homework_1().AreaOfTriangleAndCircle(firstLeg, secondLeg);
+//var areaCicle = new Homework_1().AreaOfTriangleAndCircle(radius, pi);
+//Console.WriteLine($"Площадь треугольника: {areaTriangl}");
+//Console.WriteLine($"Площадь окружности: {areaCicle}");
+
+#endregion
+#region Task_6
+//Инкапсуляция:
+//    Ребенка очень строго воспитывают в семье, но он не рассказывает об этом окружающим.
+//Наследование:
+//    Ребенок унаследовал цвет волос и глаз от родителей, но характер - его собственный.
+//Полиморфизм:
+//     Я однажды получила штраф за то, что под моими ФИО попал в дтп другой человек. 
```

Comment:
Hmm, I am not sure if this story describes current principle :(
Driving licence was pretty similar :)